avm/res/sql/managed-instance/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -106,7 +106,7 @@ module testDeployment '../../../main.bicep' = [for iteration in [ 'init', 'idem'
         name: 'customSetting'
         metricCategories: [
```

Comment:
I guess this should point at logCategories or whatever it is called?

Reply:
Fixed